aio/aio-proxy/aio_proxy/response/unite_legale_model.py

```diff
@@ -0,0 +1,130 @@
+from typing import Literal
+
+from pydantic import BaseModel
+
```

Comment:
Great file here. Love it. 

Not sure about the unite_legale prefix every where though. Is it related to your incoming association implementation ?

Reply:
It is. However, since they will be different scripts with different component fields. It is indeed redundant to use the prefix every time.